app/bin/tools/discontinue.dart

```diff
@@ -67,7 +67,7 @@ Future _set(String packageName, bool value) {
     if (p == null) {
       throw new Exception('Package $packageName does not exist.');
     }
-    p.isDiscontinued = value;
+    p.discontinuedMessage = 'Package is discontinued.';
```

Comment:
Perhaps a TODO here about allowing one to provide the message?

Reply:
Fixed with taking the rest of the arguments as a message.